terraform/ecs/main.tf

```diff
@@ -126,11 +126,23 @@ resource "aws_launch_template" "launchtemp" {
 }
 
 resource "aws_autoscaling_group" "clusterasg" {
-  name                = "clusterasg"
+  name_prefix         = "clusterasg-${module.common.testing_id}-"
```

Comment:
should the name be different? https://github.com/aws-observability/aws-otel-collector/blob/main/tools/workflow/cleaner/autoscaling/clean.go#L29